Sources/TuistKit/Extensions/Xcodeproj+Extras.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import xcodeproj
+
+extension PBXFileElement {
+    public var nameOrPath: String {
+        return name ?? path ?? ""
+    }
+}
+
+extension PBXFileElement {
+    /// File elements sort
+    ///
+    /// - Sorts elements in ascending order
+    /// - Files precede Groups
+    public static func filesBeforeGroupsSort(lhs: PBXFileElement, rhs: PBXFileElement) -> Bool {
```

Comment:
I'd unit test this one.

Reply:
👍  covered by https://github.com/tuist/tuist/pull/257/files#diff-8bb6ccb82e17648337e4021445016b98